examples/terminal_bench/test_basic.py

```
@@ -0,0 +1,145 @@
+        
+        # Create agent without calling LLM
+        agent = Agent(
+            name='test-agent',
+            instructions='Test agent for terminal tasks'
+        )
+        
+        assert agent.name == 'test-agent'
+        print("✅ Agent creation successful")
+        return True
+        
+    except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)